Week 1

BentonFraizer opened the pull request:
Добавляю все стили и шрифты к созданному в самом начале index.html.

BentonFraizer committed:
Добавлены файлы стилей, подключены шрифты

Здесь написаны файлы стилей для четырёх секции первой недели. Подключены
шрифты. Всё это запушено в ветку week_1.

BentonFraizer committed:
Изменил  пути для корректного отображения картинок

Исправил пути с формата ./.. на формат ../ для того чтобы стало видно
картинки, которые должны быть расположены на странице.

BentonFraizer committed:
Сделал один из input выбранным, т.е. checked

Как было сказано на созвоне, выбрал input с надписью "Оплата по карте" и
сделал его "выбранным" т.е. checked сразу при загрузке страницы.

BentonFraizer committed:
Изменил main.css в плане путей к картинкам

Т.к в предыдущий раз был выключен Prepros и изменения не сохранились в
main.css.

BentonFraizer committed:
В очередной раз иправил пути

Поскольку не отобразились картинки в GitHub pages ещё раз
экспериментирую с путями.

BentonFraizer committed:
Изменил вложенности в файлах .scss как в созвоне

1.Подкорректировал вложенности (уменьшил) согласно рекомендаций из
созвона;
2. Убрал классы у некоторых элементов секции "Почему", чтобы была
   возможность править данный текст из админ панели;
3. Закрасил иконку трубки телефона в белый цвет;

b-ars07 commented:
1. это должна быть контентная картинка, на групповом созвоне объяснил почему) - 
![image](https://user-images.githubusercontent.com/9081116/84834819-52ff9380-b043-11ea-9a9e-38ea6b790dbf.png)
2. нужно стили давать согласно макету (к фону) - 
![image](https://user-images.githubusercontent.com/9081116/84834903-83473200-b043-11ea-879b-2f2439f6e352.png)
3. нет ховера - 
![image](https://user-images.githubusercontent.com/9081116/84834933-965a0200-b043-11ea-8928-2e88f6e30e60.png)
4. это должен быть список из трех элементов, тоже на созвоне подробнее рассмотрели - 
![image](https://user-images.githubusercontent.com/9081116/84834969-a83ba500-b043-11ea-855b-77c95679c3f2.png)
5. лишние теги - 
![image](https://user-images.githubusercontent.com/9081116/84835049-c43f4680-b043-11ea-97c9-c47a2980753e.png)
6.  не стоит нужный type - 
![image](https://user-images.githubusercontent.com/9081116/84835066-d0c39f00-b043-11ea-89d9-10c26122054a.png)

7. отталкиваем мы сверху-вниз - 
![image](https://user-images.githubusercontent.com/9081116/84835106-e46f0580-b043-11ea-80c3-cc0a1cb5b5b8.png)

BentonFraizer committed:
Исправлены замечания наставника

Конкретнее:
1. Картинка с батончиками секции hero сделана контрентной, т.е. не через
background;
2. Заменён цвет фона секции с отзывами;
3. Сделан hover-эффект на элементах-переключателях отзывов;
4. В секции с отзывами сформирован список из 3-х елементов-отзывов, 2 из
которых временно закомментированы;
5. Убраны лишние теги;
6. Выставлены нужные типы для элементов input, дополнительно скрыты
стрелочки у элементов с типом number (!не исправлено для браузера FireFox);
7. Кнопки секции с доставкой отодвинуты елементами, расположенными
сверху.

BentonFraizer committed:
Поправлен путь до svg-картинки checkbox

BentonFraizer committed:
Не в том месте поправил путь в первый раз

b-ars07 commented:
1. список не может быть из одного элемента, все элементы должны быть сверстаны и находится в разметке
![image](https://user-images.githubusercontent.com/9081116/85587349-c5c7ca80-b652-11ea-8955-d5054f80434f.png)

BentonFraizer committed:
Поправлен список отзывов

Раскоментированы скрытые комментарии в index.html
И сразу же были скрыты при помощи CSS.

BentonFraizer committed:
Добавлен доп класс с модифитарором к отзывам
